pkgs/applications/misc/1password-gui/beta.nix

```diff
@@ -58,7 +59,8 @@ in stdenv.mkDerivation rec {
         sha256 = "0vqrcwn5y350g91w3kh8n43gw21kck1cwim92dw9i0xxxch91hrg";
       };
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper wrapGAppsHook ];
```

Comment:
Isn't this now double wrapping?

Reply:
The beta derivation is missing the `dontWrapGApps` line that default has.

Reply:
Fixed :+1: